examples/v2/gke/README.md

```diff
@@ -146,30 +146,62 @@ If you manage certain resource types (`ClusterRole`, etc), you will need to gran
 
 to  the service account.
 
-## Deploying other types
+## Deploying CustomResourceDefinitions
 
-If you want to deploy any other k8s artifact, create a collection reference and apply the [k8s API specifications](https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json) for that resource.
+If you want to deploy other dynamic k8s artifacts such as [CustomResourceDefinitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions), define the collection first for that resource and then the object itself.
 
-For example, to _define_  `CustomResourceDefinitions`,
+For example, for `CustomResourceDefinitions`, create a cluster that supports [CustomResourcePublishOpenAPI](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#publish-validation-schema-in-openapi-v2).  At the time of writing `11/6/19`, you need a GKE cluster version of atleast `1.14.7` plus Alpha feature gate enabled:
```

Comment:
We know already that 1.15, whenever it comes out, will have CRDs available in beta, so no alpha feature gate required. Thus I think we should amend this to be something like "plus Alpha feature gate enabled if you are below 1.15".

Reply:
done